config/clients/java/template/Java/ServerVariable.mustache

```diff
@@ -0,0 +1,23 @@
+package {{invokerPackage}};
```

Comment:
Delete this file and ignore it so it doesn't get generated

Reply:
Removed in [6f9804d](https://github.com/openfga/sdk-generator/pull/158/commits/6f9804d21724edeff04096b82f2a999ab7a75144) (Also a related `ServerConfiguration` class that uses ServerVariable and is not used anywhere else)